kubernetes-deployment/test/test_maintainlib_common.py

```diff
@@ -119,6 +119,40 @@ def test_package(self):
 
 
 
+    def test_ipv4_address_validation_correct(self):
+
+        addr = "128.0.0.1"
+        self.assertTrue(common.ipv4_address_validation(addr))
+
+
+
+    def test_ipv4_address_validation_wrong(self):
+
+        addr1 = "128.0.0.x"
+        self.assertFalse(common.ipv4_address_validation(addr1))
+
+        addr2 = "256.0.0.0"
+        self.assertFalse(common.ipv4_address_validation(addr2))
+
+
+
+    def test_port_validation_correct(self):
+
+        port1 = 22
+        self.assertTrue(common.port_validation(port1))
+
+        port2 = "232"
```

Comment:
please also try 65536, -22. also should we consider port 0 as invalid port (anyport)?

Reply:
0 is valid.

Reply:
Both port and ipv4 address checking should be treated as format validation, not definition validation.